modules/runlevel.c

```diff
@@ -81,8 +81,30 @@ static void shutdown(dsme_runlevel_t runlevel)
            runlevel == DSME_RUNLEVEL_REBOOT   ? "Reboot"   :
                                                 "Malf");
 
+  /* If we have systemd, use systemctl commands */
```

Comment:
Should this also go in `#ifdef DSME_SYSTEMD_ENABLE` ?

Reply:
This was done for simplicity. And I tried to stick the decisions made by Mer developers to make our code closer to what they have.

If you don't like this part, I can rewrite it.

Reply:
No, it's good to keep it close to what Mer project does. Maybe one day we can just use their mer version.